### PR TITLE
collect logs of loader

### PR DESCRIPTION
**Pull request by amoskong**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3197,6 +3197,16 @@ def kill_gemini_thread(self):
                 if kill_result.exit_status != 0:
                     self.log.error('Terminate gemini on node %s:\n%s', loader, kill_result)
 
+    def collect_logs(self, storing_dir):
```
Comment (fruch) [inline sdcm/cluster.py]: @aleksbykov please consider this for you log collecting PR...

**Comment by fruch** [comment]
@amoskong funny thing that we didn't noticed that until now. we should get used to investigate via uploaded logs, and direct on the jenkins workers

**Comment by amoskong** [comment]
> @amoskong funny thing that we didn't noticed that until now. we should get used to investigate via uploaded logs, and direct on the jenkins workers

We can live by grep sct.log, but it's pain sometimes ;-)

**Comment by amoskong** [comment]
http://jenkins.scylladb.com/view/scylla-3.1/job/scylla-3.1/view/scylla-3.1-test/job/rolling-upgrade/job/rolling-upgrade-ubuntu18.04/80/execution/node/41/log/

IN this job, the loader logs were uploaded to s3:
- https://cloudius-jenkins-test.s3.amazonaws.com/a66b5b4f-e8f2-42e8-af2a-8f8849fab6d4/rolling-upgrade-upgrade--ubuntu-bio-loader-set-a66b5b4f.zip

But I didn't see it in hydra investigate. Do we need to add log type in elasticsearch?

```
$ hydra investigate show-logs e7ed3046-2437-4989-b21a-c85eb42f1a49

+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                 Log links for testrun with test id e7ed3046-2437-4989-b21a-c85eb42f1a49                                                  |
+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
| Log type              | Link                                                                                                                                             |
+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
| events                | https://cloudius-jenkins-test.s3.amazonaws.com/e7ed3046-2437-4989-b21a-c85eb42f1a49/events_log.zip                                               |
| job                   | https://cloudius-jenkins-test.s3.amazonaws.com/e7ed3046-2437-4989-b21a-c85eb42f1a49/job_log.zip                                                  |
| monitoring_data_stack | https://cloudius-jenkins-test.s3.amazonaws.com/e7ed3046-2437-4989-b21a-c85eb42f1a49/monitoring_data_stack_branch-2.4_2.3.tar.gz                  |
| db-cluster            | https://cloudius-jenkins-test.s3.amazonaws.com/e7ed3046-2437-4989-b21a-c85eb42f1a49/rolling-upgrade-upgrade--ubuntu-xen-db-cluster-e7ed3046.zip  |
| monitor-set           | https://cloudius-jenkins-test.s3.amazonaws.com/e7ed3046-2437-4989-b21a-c85eb42f1a49/rolling-upgrade-upgrade--ubuntu-xen-monitor-set-e7ed3046.zip |
+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
```

**Comment by bentsi** [comment]
@amoskong please add it to hydra too in `sdcm.utils.common.list_logs_by_test_id`

**Comment by amoskong** [comment]
> @amoskong please add it to hydra too in `sdcm.utils.common.list_logs_by_test_id`

added, it works. Thanks.

**Comment by bentsi** [comment]
cc: @aleksbykov please track all the changes/fixes and add to collect logs PR if needed

**Pull request merged**